### PR TITLE
[v0.5.2] Cherry-picking licenses from development

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1930

* **Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,13 +31,14 @@
...
+    "test": "jest --coverage ./tests/*",
+    "check-licenses": "apache2-license-checker"
   },
   "dependencies": {
-    "@dojot/adminkafka": "0.0.1-alpha.9",
-    "@dojot/dojot-module": "0.0.1-beta.9",
-    "@dojot/dojot-module-logger": "0.0.1-alpha.5",
-    "@dojot/healthcheck": "^0.0.1-alpha",
+    "@dojot/adminkafka": "0.2.0",
+    "@dojot/dojot-module": "0.1.0",
```
Comment (rascaraficci) [inline package.json:39]: I think it should be 0.1.1. If I'm not wrong the version 0.1.0 is broken.

**Review by rascaraficci — CHANGES_REQUESTED** [review]
I left a comment.

File: package.json (hunk 1 above)
Comment (mprevide) [reply package.json:39]: Ok, will create o pr to dev too

**Review by thiagodpf — APPROVED** [review]

**Review by rascaraficci — APPROVED** [review]
It looks OK!

**Pull request merged**